web/modules/custom/welcome/welcome.permissions.yml

```diff
@@ -0,0 +1,4 @@
+administer hello_world:
```

Comment:
Where is it used?

Reply:
I did not use it previously but now i have used it.